tests/example/stable.go

```diff
@@ -0,0 +1,201 @@
+package example
+
+import (
+	"path/filepath"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	examplev1 "github.com/openshift/api/example/v1"
+
+	apiextensionsv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/utils/pointer"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/envtest"
+	"sigs.k8s.io/controller-runtime/pkg/envtest/komega"
+)
+
+var _ = Describe("[Stable] Example API", func() {
```

Comment:
These all appear to follow a pattern of either
1. create
2. expected error

or
1. create 
2. expect success

or
1. create
2. expect success
3. update
4. failure message


Given the suggested fanout from these integration tests, I think your time would be well spent to allow directories to be created with content with files like create,  expected_create_failure, update_1, expected_update_1_failure and auto-run the tests based on those.  That will help you get traction for your common cases.